File: data/code_pairs/fortran/2035.f90
program main
    implicit none
    integer :: intVar
    real :: floatVar
    character(len=10) :: strVar
    character(len=10) :: str
    integer :: num

    ! Variable declarations
    intVar = 10
    floatVar = 20.5
    strVar = 'Hello'

    ! Arithmetic operations
    intVar = intVar + 5
    floatVar = floatVar * 2.0

    ! Character to integer conversion
    str = '123'
    read(str, *) num

    ! Integer to character conversion
    write(str,'(I10)') num

end program main

File: data/code_pairs/cpp/2035.cpp
#include <iostream>
#include <string>
#include <sstream>

int main() {
    int intVar = 10;
    float floatVar = 20.5;
    std::string strVar = "Hello";

    // Arithmetic operations
    intVar = intVar + 5;
    floatVar = floatVar * 2.0;

    // Character to integer conversion
    std::string str = "123";
    int num;
    std::stringstream(str) >> num; // String to integer

    // Integer to character conversion
    str = std::to_string(num); // Integer to string

    return 0;
}